### PR TITLE
Refactor BaseConnectorTest#testComment* to reduce queries

### PR DESCRIPTION
**Pull request by hashhar**
## Description

This helps with BigQuery for example since number of metadata operations per second is limited.
Other tests also benefit with being slightly faster.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

### CODE REVIEW

**Review by raunaqmorarka — APPROVED** [review]

**Review by wendigo — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java [hunk 1]
```diff
@@ -3578,19 +3578,17 @@ public void testCommentTable()
                     .skippingTypesCheck()
                     .containsAll("VALUES ('" + table.getName() + "', 'new comment')");
 
+            // comment deleted
+            assertUpdate("COMMENT ON TABLE " + table.getName() + " IS NULL");
+            assertThat(getTableComment(catalogName, schemaName, table.getName())).isEqualTo(null);
+
             // comment updated
             assertUpdate("COMMENT ON TABLE " + table.getName() + " IS 'updated comment'");
             assertThat(getTableComment(catalogName, schemaName, table.getName())).isEqualTo("updated comment");
```
Comment (ebyhr) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:3587]: This block might be a little misleading now. "updated comment" was correct in the previous code, but it's a new comment now.

Comment (ebyhr) [reply testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:3587]: I'm not saying about test coverage. I mean `null -> non-null` isn't "updated comment"  and it's a new comment in strict.

Comment (hashhar) [reply testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:3587]: I'll update the comment to something like `comment added to verify setting empty comment`? Or should I remove the assertion completely?

Comment (hashhar) [reply testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:3587]: @ebyhr I'll update the comment to something like `comment added to verify setting empty comment`? Or should I remove the assertion completely?

Comment (hashhar) [reply testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java]: updated the comment to be more true.

**Pull request merged**